conf/production.yml

```diff
@@ -39,9 +39,9 @@
     - name: Set common facts
       set_fact:
         domain1_name: "pori.prod.wunder.io"
-        domain2_name: "pori.fi"
-        domain3_name: "visitpori.fi"
-        domain5_name: "businesspori.fi"
+        domain2_name: "pori.prod.wunder.io"
```

Comment:
This needs to be provisioned via ansible

Reply:
Added related comment to the PR description.